README.md

```diff
@@ -33,70 +39,83 @@ The content of the output folder could look as follows:
 
 For a list of options see:
 
-    $ meteor-build-client --help
+```shell
+meteor-build-client --help
+```
 
 ### Passing a settings.json
 
 You can pass an additional settings file using the `--settings` or `-s` option:
 
-    $ meteor-build-client ../myOutputFolder -s ../settings.json
+```shell
+meteor-build-client ../myOutputFolder -s ../settings.json
```

Comment:
This has also been tested using the latest docs build for https://github.com/jankapunkt/meteor-blaze-bs4

Reply:
👍